test/integration/etcd/data.go

```diff
@@ -342,26 +342,6 @@ func GetEtcdStorageDataForNamespaceServedAt(namespace string, v string, removeAl
 		},
 		// --
 
-		// k8s.io/kubernetes/pkg/apis/flowcontrol/v1beta3
-		gvr("flowcontrol.apiserver.k8s.io", "v1beta3", "flowschemas"): {
-			Stub:              `{"metadata": {"name": "fs-2"}, "spec": {"priorityLevelConfiguration": {"name": "name1"}}}`,
-			ExpectedEtcdPath:  "/registry/flowschemas/fs-2",
-			ExpectedGVK:       gvkP("flowcontrol.apiserver.k8s.io", "v1", "FlowSchema"),
-			IntroducedVersion: "1.26",
-			RemovedVersion:    "1.32",
-		},
-		// --
-
-		// k8s.io/kubernetes/pkg/apis/flowcontrol/v1beta3
-		gvr("flowcontrol.apiserver.k8s.io", "v1beta3", "prioritylevelconfigurations"): {
-			Stub:              `{"metadata": {"name": "conf4"}, "spec": {"type": "Limited", "limited": {"nominalConcurrencyShares":3, "limitResponse": {"type": "Reject"}}}}`,
-			ExpectedEtcdPath:  "/registry/prioritylevelconfigurations/conf4",
-			ExpectedGVK:       gvkP("flowcontrol.apiserver.k8s.io", "v1", "PriorityLevelConfiguration"),
-			IntroducedVersion: "1.26",
-			RemovedVersion:    "1.32",
```

Comment:
@jpbetz @siyuanfoundation @Jefftree can 1.34 emulate 1.31 or only 1.32+?

Reply:
1.34 can emulate 1.31 (n-3). We should hold off on removing this until 1.35 then.

Reply:
The api version was removed at 1.32, but the admin can re-serve it via using the env `KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE`. so we have to postpone it until 1.36. Is it right? cc @Jefftree

Reply:
https://github.com/carlory/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/deleted_kinds.go#L150

Reply:
cc @liggitt